assets/src/block-validation/store.js

```diff
@@ -15,11 +15,16 @@ import {
 
 export const BLOCK_VALIDATION_STORE_KEY = 'amp/block-validation';
 
+const SET_IS_FETCHING_ERRORS = 'SET_IS_FETCHING_ERRORS';
+const SET_IS_POST_DIRTY = 'SET_IS_POST_DIRTY';
 const SET_IS_SHOWING_REVIEWED = 'SET_IS_SHOWING_REVIEWED';
+const SET_REVIEW_LINK = 'SET_REVIEW_LINK';
 const SET_VALIDATION_ERRORS = 'SET_VALIDATION_ERRORS';
 
 export const INITIAL_STATE = {
 	ampCompatibilityBroken: false,
+	isPostDirty: false,
+	isFetchingErrors: false,
```

Comment:
This should be `true` if it is not a brand new post.

Reply:
Well, actually the initial state of this flag should be `false` since the errors fetching was not triggered just yet (the `INITIAL_STATE` is not meant to trigger actions but rather to represent the actual state).

The initial errors fetching is [triggered in the `useValidationErrorStateUpdates` hook](https://github.com/ampproject/amp-wp/blob/feature/2069-async-url-validation/assets/src/block-validation/use-validation-error-state-updates.js#L108-L115) (which as noted in [another comment](https://github.com/ampproject/amp-wp/pull/5741#discussion_r583600344) currently has a bug – the `isFetchingErrors` flag is not set there; it's going to be addressed by bba583a).